app/models/post.rb

```diff
@@ -33,65 +33,36 @@ def self.inherited(child)
           indexes :organization_id, type: :integer
         end
       end
-
-    end
-  end
-
-  def index_document
-    __elasticsearch__.index_document
-  end
-
-  # pass member when doing bulk things
-  def update_or_delete_document(member=nil)
-    member ||= self.member
-    if active && member.try(:active)
-      __elasticsearch__.update_document rescue __elasticsearch__.index_document
-    else
-      __elasticsearch__.delete_document rescue nil
     end
   end
 
-  def delete_document
-    __elasticsearch__.delete_document rescue nil
-  end
-
-
-  def as_indexed_json(*)
-    as_json(only: [:title, :description, :tags, :organization_id])
-  end
-
   attr_reader :member_id
 
   belongs_to :category
-  delegate :name, to: :category, prefix: true, allow_nil: true
+
   belongs_to :user
   belongs_to :organization
   belongs_to :publisher, class_name: "User", foreign_key: "publisher_id"
   # belongs_to :member, class_name: "Member", foreign_key: "user_id"
-
   has_many :user_members, class_name: "Member", through: :user, source: :members
-
   has_many :transfers
   has_many :movements, through: :transfers
 
-  default_scope -> { order("posts.updated_at DESC") }
+  delegate :name, to: :category, prefix: true, allow_nil: true
 
+  default_scope { order("posts.updated_at DESC") }
```

Comment:
I prefer to keep all the arguments of `scope` being lambdas (not blocks), just to have one less thing to think about. There is not much difference.